src/trans/gpu/internal/trgtol_mod.F90

```diff
@@ -1482,13 +1548,12 @@ SUBROUTINE TRGTOL(PGLAT,KF_FS,KF_GP,KF_SCALARS_G,KVSET,KPTRGP,&
 ! when run as gpnorm on cpu needs to avoid this update call
 IF(.NOT.LGPNORM) THEN
 #ifdef ACCGPU
-  !$ACC UPDATE DEVICE(PGLAT)
-#endif
-#ifdef OMPGPU
-  !$OMP TARGET UPDATE TO(PGLAT)
+!! All data is updated on device already. PJM 9/05/2023
+!  !$ACC UPDATE DEVICE(PGLAT)
 #endif
 #ifdef ACCGPU
-  !$ACC WAIT
+!! I don't understand the need for this. PJM 9/05/2023
+!  !$ACC WAIT
```

Comment:
Hmm there are no `ASYNC` statements in this file. Maybe this is just leftover code?

Reply:
Not sure what to do with this code. Anyone else know why we need this?